UI-Session-3-Assignment/index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Home</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous" />
+</head>
+
+<body>
+    <!-- header for all web pages of this site contain logo-image ,menu and social links of facebook instagram and google plus-->
+    <header id="header">
+        <!--company logo-->
+        <img class="logo" src="images\logo.png" alt="Logo image">
+        <!-- menu unordered list-->
+        <ul class="menu">
+            <li> <a href="">Home</a> </li>
+            <li> <a href="">Features</a> </li>
+            <li> <a href="">Review</a> </li>
+            <li> <a href="">Download</a> </li>
+        </ul>
+        <!-- social links of company-->
+        <div class="social-links">
+            <a href=""> <i class="fab fa-facebook-f"></i> </a>
+            <a href=""> <i class="fab fa-instagram"></i> </a>
+            <a href=""> <i class="fab fa-google-plus-g"></i> </a>
+        </div>
+    </header>
+
+    <!-- cover section containing Download and feature option -->
+    <section id="cover-section">
+        <div>
+            <h1>Get the most fun weather app</h1>
+            <a href="">Download</a>
+            <a href="">Features</a>
+        </div>
+        <div>
+            <img src="images\phone1.png" alt="Cloudy image">
+        </div>
+    </section>
+
+    <section id="container">
+        <!--Feature section containing features of  -->
+        <section id="feature-section">
+            <h1 class="heading">Perfect Features</h1>
+            <h5 class="sub-heading">Only necessary</h5>
+            <section id="features-headline">
+                <!-- feature 1-->
+                <div class="feature" id="feature1">
+                    <img src="images\feature1.png" alt="Usability image">
+                    <div>
+                        <h1>Usability</h1>
+
+                        <h5>Sometimes the simplest things are the hardest to find. So we created a nwe line for everyday life.</h5>
```

Comment:
Do not to use heading tag here. you can use a paragraph or any other tag. Because in the given layout it is not shown as a heading.
Same for the others.

Reply:
Done